runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/harness/StreamingWorkerStatusReporter.java

```diff
@@ -81,6 +83,12 @@ public final class StreamingWorkerStatusReporter {
   private final ScheduledExecutorService globalWorkerUpdateReporter;
   private final ScheduledExecutorService workerMessageReporter;
 
+  // PerWorkerMetrics are sent on the WorkerMessages channel, and are sent one in every
+  // perWorkerMetricsUpdateFrequency RPC call. If 0, PerWorkerMetrics are not reported.
+  long perWorkerMetricsUpdateFrequency = 0L;
```

Comment:
if this never changes lets make it private and final and initialize in the constructor

you can modify the constructor to take in this instead of passing it into start() and have start() take no params.